examples/tp/hello_tp.py

```diff
@@ -39,13 +39,19 @@
 # Can't live without numpy
 import numpy
 
+# izip for maximum efficiency
+from itertools import izip as zip, count
+
 # here we choose TP implementation: (uncomment one only)
-# This is the class corresponding to the C++ optimized Temporal Pooler (default)
-from nupic.research.TP10X2 import TP10X2 as TP
+# This is the class corresponding to the C++ optimized Temporal Pooler(default)
+# from nupic.research.TP10X2 import TP10X2 as TP
 # This is simple implementation in Python
-#from nupic.research.TP import TP as TP
+
+from nupic.research.temporal_memory import TemporalMemory as TM
```

Comment:
if there's already c++ bindings for TM (?) you could show how to import them here as well, like it was done with TP.